model/src/main/kotlin/DependencyGraphNavigator.kt

```diff
@@ -49,9 +51,13 @@ class DependencyGraphNavigator(
         //       they manage, for example that "GradleInspector" manages "Gradle" projects.
         val managers = graphs.keys.filter { it.startsWith(project.id.type) }
 
-        val manager = requireNotNull(managers.singleOrNull()) {
-            "All of the $managers managers are able to manage '${project.id.type}' projects. Please enable only one " +
-                "of them."
+        val manager = managers.first()
+
+        if (managers.size > 1) {
+            logger.error {
+                "All of the $managers managers are able to manage '${project.id.type}' projects. Please enable only " +
```

Comment:
Should it say "All of the managers, $managers, are able to..."?

Reply:
I didn't want to change too much compared to the existing wording, and mind you that `$managers` actually expands to e.g. `[Gradle, GradleInspector]`; so with the square brackets I believe the current wording reads a bit better.